async port scanner.py

```diff
@@ -0,0 +1,48 @@
+"""
+Async port scanner.
+"""
+
+import time
+import asyncio
+
+
+async def connect_socket(ip: str, port: int):
+    """
+    Connect to given IP and port
+
+    :param ip: The ip for the socket to connect.
+    :param port: The port for the socket to connect.
+    """
+    try:
+        await asyncio.open_connection(ip, port)
+        print(f"port {port} found!")
+    except WindowsError:
```

Comment:
The error object is too general. Isn't there a connection-specific-error?

Reply:
I didn't find any specific error